### ISSUE TITLE
doc, not updated to recent version 6.0

### ISSUE THREAD

**Issue by WIZARDspace** [hemilabs/heminetwork#335]
### Confirmation

- [x] I have checked for similar issues.

### Describe the bug

the DOC, is not updated , old and new users who are building on the hemi network still see the old version 0.5.0. kindly update the DOC to the recent version ![Image](https://github.com/user-attachments/assets/6daa9383-71f3-4fc8-b2e8-a15a26f3f9c2)

### Expected behaviour

the  CLI is not mining keystone due to this old version 

### Environment

Version: 
Operating System: 
Architecture:

**Comment by lpopmo** [comment]
yes i am also facing the same problem the file doesn't run in the terminal after extracting and pasting the file in the terminal it says the file is not recognizable
